engines/pytorch/pytorch-engine/src/test/java/ai/djl/pytorch/engine/PtNDArrayExTest.java

```diff
@@ -0,0 +1,49 @@
+package ai.djl.pytorch.engine;
+
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDList;
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.types.DataType;
+import ai.djl.ndarray.types.Shape;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class PtNDArrayExTest {
+    public static void main(String[] args) {
+        PtNDArrayExTest ptNDArrayExTest = new PtNDArrayExTest();
+        ptNDArrayExTest.testMultiBoxPrior();
+    }
+
+    @Test
+    public void testMultiBoxPrior() {
```

Comment:
You can use MultiBoxPriorTest to test for PyTorch as well.

Reply:
Cool - deleted.